src/main/java/io/pravega/connectors/flink/util/StreamId.java

```diff
@@ -1,77 +0,0 @@
-/**
- * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- */
-package io.pravega.connectors.flink.util;
-
-import org.apache.commons.lang3.StringUtils;
-
-import java.util.Objects;
-
-/**
- * Captures the fully qualified name of a stream. The convention to represent this as a
- * single string is using [scope]/[stream].
- */
-public class StreamId {
```

Comment:
The description says that this is being deprecated, but the class is actually being removed rather than marked as deprecated.